### PR TITLE
[IMP] l10n_cl: Update doc for Odoo 14 Accounting

### PR DESCRIPTION
**Pull request by ren-odoo**
Update the elements and configuration that changes from V13 to V14.  
- Remove sequences. 
- Add new CAF menu.

### CODE REVIEW

File: accounting/fiscal_localizations/localizations/chile.rst [hunk 1]
```diff
@@ -287,74 +287,50 @@ applicable to Purchase and Sales journals that can be related to the
 different sets of document types available in Chile. By default, all the sales journals created will
 use documents. 
   
-.. image:: media/Chile10.png
```
Comment (AntoineVDV) [inline accounting/fiscal_localizations/localizations/chile.rst:290]: Is this file still used somewhere?

File: accounting/fiscal_localizations/localizations/chile.rst [hunk 2]
```diff
@@ -287,74 +287,50 @@ applicable to Purchase and Sales journals that can be related to the
...
-*********
-
-When a Journal is created we have two options with document types sequences. 
-
-1. Create new sequences: In this case, Odoo will create new sequences for the most common document
-   types. 
-2. Use existent sequence: If the sequence should be shared between several journals, the user can
-   add manually sequences previously created.  
-
-.. image:: media/Chile11.png
```
Comment (AntoineVDV) [inline accounting/fiscal_localizations/localizations/chile.rst:304]: Is this file still used somewhere?

File: accounting/fiscal_localizations/localizations/chile.rst [hunk 3]
```diff
@@ -287,74 +287,50 @@ applicable to Purchase and Sales journals that can be related to the
...
 
 	
 Configuration
-^^^^^^^^^^^^^
-
-Once you have the CAF files you need to associate them with journal sequence in Odoo,
-after you create the sequence for the first time the will be in red, which indicates
-that no CAF has been configured for that particular sequence:
-
-.. image:: media/Chile12.png
```
Comment (AntoineVDV) [inline accounting/fiscal_localizations/localizations/chile.rst:335]: Is this file still used somewhere?

File: accounting/fiscal_localizations/localizations/chile.rst [hunk 4]
```diff
@@ -287,74 +287,50 @@ applicable to Purchase and Sales journals that can be related to the
...
-4. Save the CAF.
-5. Save the sequence. 
+Once you have the CAF files you need to associate them with a document type in Odoo,
+in order to add a CAF, just follow these steps:
+ 
+1. Access to :menuselection:`Accounting --> Settings --> CAF`
+2. Upload the file.
+3. Save the CAF.
 
-.. image:: media/Chile13.gif
```
Comment (AntoineVDV) [inline accounting/fiscal_localizations/localizations/chile.rst:347]: Is this file still used somewhere?

File: accounting/fiscal_localizations/localizations/chile.rst [hunk 5]
```diff
@@ -287,74 +287,50 @@ applicable to Purchase and Sales journals that can be related to the
...
+.. image:: media/Chile39.gif
```
Comment (AntoineVDV) [inline accounting/fiscal_localizations/localizations/chile.rst:323]: This file is WAY too large (3.5 MB) in comparison with its added value.
In git, a repository can only grow in size, never shrink. We must be careful not to add unnecessary large files as every single update made to those files in fact duplicates them. This doesn't scale at all and we are still paying the price of such mistakes in the odoo/odoo repository whose weight is now measured in GB.

File: accounting/fiscal_localizations/localizations/chile.rst [hunk 6]
```diff
@@ -287,74 +287,50 @@ applicable to Purchase and Sales journals that can be related to the
...
+Once is loaded the status changes to "In Use", at this moment when a transaction is used
+for this document type the invoice number will take the first folio in the sequence. 
 
-As a result, the sequence color changes to black and the column Next number and quantity
-available are updated:
+.. important::
+   In case you have use some folios in you previuos system, make sure you set the next valid
+   folio when the first transation is created. 
 
-.. image:: media/Chile13.png
```
Comment (AntoineVDV) [inline accounting/fiscal_localizations/localizations/chile.rst:355]: Is this file still used somewhere?

File: accounting/fiscal_localizations/localizations/chile.rst [hunk 7]
```diff
@@ -531,25 +507,24 @@ you might have and which is the related solution.
...
 
 Credit Notes
 ------------
 
 When a cancellation or correction is needed over a validated invoice, a credit note must be
-generated. It is important to consider that a CAF file is required for the Credit Note sequence,
+generated. It is important to consider that a CAF file is required for the Credit Note,
 which is identified as document 64 in the SII.  
 
-.. image:: media/Chile25.png
```
Comment (AntoineVDV) [inline accounting/fiscal_localizations/localizations/chile.rst:546]: Is this file still used somewhere?

**Review by AntoineVDV — CHANGES_REQUESTED** [review]
Please update your commit message to follow the guidelines (see `commit_template.txt`).
A suitable title could be: "[IMP] l10n_cl: update doc for Odoo 14 accounting"

File: accounting/fiscal_localizations/localizations/chile.rst [hunk 8]
```diff
@@ -287,74 +287,50 @@ applicable to Purchase and Sales journals that can be related to the
...
+3. Save the CAF.
 
-.. image:: media/Chile13.gif
+.. image:: media/Chile39.gif
    :align: center
-   :alt: Steps to add a new CAF on sequences.
+   :alt: Steps to add a new CAF.
 
+Once is loaded the status changes to "In Use", at this moment when a transaction is used
+for this document type the invoice number will take the first folio in the sequence.
```
Comment (jcs-odoo) [inline accounting/fiscal_localizations/localizations/chile.rst:328]:
```suggestion
Once loaded, the status changes to *In Use*. At this moment, when a transaction is used
for this document type, the invoice number takes the first folio in the sequence. 
```

**Review by jcs-odoo — CHANGES_REQUESTED** [review]
The rst is good!

There are a few modifications needed:

- please change the commit message to something like `[IMP] localizations: Chile update`
- do not mention the version number in the content.
- remove the unused files from the repository. All unused png must be deleted. But no worries, they remain in the previous versions.
- do you use PNGQUANT to shrink your image files? Everything is explained [here](https://www.odoo.com/documentation/user/14.0/contributing/documentation/introduction_guide.html)

Let me know if you need assistance with all this. 👍 

Cheers,
Jonathan

File: accounting/fiscal_localizations/localizations/chile.rst (hunk 1 above)
Comment (ren-odoo) [reply accounting/fiscal_localizations/localizations/chile.rst:290]: No, it has been removed in the last commt

File: accounting/fiscal_localizations/localizations/chile.rst (hunk 2 above)
Comment (ren-odoo) [reply accounting/fiscal_localizations/localizations/chile.rst:304]: No, it has been removed in the last commt

File: accounting/fiscal_localizations/localizations/chile.rst [hunk 9]
```diff
@@ -287,74 +287,50 @@ applicable to Purchase and Sales journals that can be related to the
...
+A CAF (Folio Authorization Code) is required, for each document type that is issued to your
+customers, the CAF is a file the SII provides to the Emisor with the folio/sequence authorized 
+for the electronic invoice documents.
 
 Your company can make several requests for folios and obtain several CAFs, each one associated
-with different ranges of folios. In this way, you could order folios per the different
-segmentations you have in your journals (example per office) thus avoiding distributing a range
-of folios. In this way, each branch would have its private key of the respective CAF.
+with different ranges of folios. In V14 the CAF are shared within all the journals, this means
+that you only need one active CAF per document type and it will be applied on all journals.
```
[the remark this answers is not in the document]
Comment (ren-odoo) [reply accounting/fiscal_localizations/localizations/chile.rst]: Got it, it has been corrected in the last commit.

File: accounting/fiscal_localizations/localizations/chile.rst (hunk 3 above)
Comment (ren-odoo) [reply accounting/fiscal_localizations/localizations/chile.rst:335]: No, it has been removed in the last commit

File: accounting/fiscal_localizations/localizations/chile.rst (hunk 4 above)
Comment (ren-odoo) [reply accounting/fiscal_localizations/localizations/chile.rst:347]: No, it has been removed in the last commit

File: accounting/fiscal_localizations/localizations/chile.rst (hunk 5 above)
Comment (ren-odoo) [reply accounting/fiscal_localizations/localizations/chile.rst]: Got it, I removed it and switched it for a png image.

[the remark this answers is not in the document]
Comment (ren-odoo) [reply accounting/fiscal_localizations/localizations/chile.rst]: Got it, I removed it and switched it for a png image.

File: accounting/fiscal_localizations/localizations/chile.rst (hunk 6 above)
Comment (ren-odoo) [reply accounting/fiscal_localizations/localizations/chile.rst:355]: No, it has been removed in the last commit

File: accounting/fiscal_localizations/localizations/chile.rst (hunk 7 above)
Comment (ren-odoo) [reply accounting/fiscal_localizations/localizations/chile.rst:546]: No, it has been removed in the last commit

**Comment by ren-odoo** [comment]
> The rst is good!
> 
> There are a few modifications needed:
> 
> * please change the commit message to something like `[IMP] localizations: Chile update`
> * do not mention the version number in the content.
> * remove the unused files from the repository. All unused png must be deleted. But no worries, they remain in the previous versions.
> * do you use PNGQUANT to shrink your image files? Everything is explained [here](https://www.odoo.com/documentation/user/14.0/contributing/documentation/introduction_guide.html)
> 
> Let me know if you need assistance with all this. 👍
> 
> Cheers,
> Jonathan

Thank you @jcs-odoo I took into consideration all your feedback in the last commits, I think it should be ok now.  Regardin PNGquant yes I used to as instructed. 

Let me know if there is anything else to be corrected. 

Thank you.

**Review by AntoineVDV — CHANGES_REQUESTED** [review]
The content LGTM on the tech side :+1: 
The commits still need to be squashed but then it's good to go!

**Review by jcs-odoo — APPROVED** [review]

**Comment by jcs-odoo** [comment]
Seems good to me!
I already approved the PR so that it can directly be merged once you've done the squash Antoine mentioned.

Cheers :-)  
Jonathan

**Comment by ren-odoo** [comment]
Hi @jcs-odoo  @AntoineVDV  the squas has been done. 

Regards.

**Review by AntoineVDV — APPROVED** [review]

**Comment by AntoineVDV** [comment]
Hi @ren-odoo 

Pro-tip for commit squashing: if the titles or commit messages of some commits are not relevant (such as in this case), you should dismiss them when squashing the commits into the first one.

For recall, you can squash commits with `git rebase -i 14.0` and then change the command before each commit, except the first one, from "pick" to either "fixup" (if you want to discard the commit message) or "squash".

**Pull request merged**